deploy/terraform/variables.tf

```diff
@@ -500,6 +500,12 @@ variable "memory" {
   default = 8192
 }
 
+variable "tcp_network_protocol" {
```

Comment:
Does this make sense to be a variable?  I can't see this wanting to change 🤔.

Reply:
yes, I think it is still makes sense as it looks more concise. Also, in some places you can spot some upper case strings in some other ones lower cases hence having it as a variable will be more concise.